From 608b0352e301e6e3c6a7d239b938a3b6af2c031d Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 18 Apr 2021 22:47:24 +0300
Subject: [PATCH] [SW-0] gaudi: security: align offsets naming

(Re)use the same naming as used in the rest of the code, so the RRs
offsets settings will have the same naming as the actual image.

Change-Id: I42af62c5b6e02439f8586dd2ff640654ee5ccd20
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/security.c b/gaudi/security.c
index c1e3b2c..e67cb14 100644
--- a/gaudi/security.c
+++ b/gaudi/security.c
@@ -1611,9 +1611,8 @@
 
 #define SRAM_SIZE	0x1400000	/* 20MB */
 #define SRAM_END	(SDRAM_BASE + SRAM_SIZE)
-#define BOOT_FIT_START	(SDRAM_BASE + BOOT_FIT_SRAM_OFFSET)
-#define BOOT_FIT_MAX_SZ	0x100000	/* 1MB */
-#define BOOT_FIT_END	(BOOT_FIT_START + BOOT_FIT_MAX_SZ)
+#define FIT_IMG_MAX_SZ	0x100000	/* 1MB */
+#define FIT_IMG_END	(FIT_IMG_BASE + FIT_IMG_MAX_SZ)
 
 enum preboot_sec_rr_names {
 	SRAM_BGN_TO_VERS = 0,
@@ -1644,16 +1643,16 @@
 	[VERS_END_TO_IMG] = {
 		.rr = PCIE_DYN_RR_SRAM_R2,
 		.min = 0,	/* version end */
-		.max = BOOT_FIT_START,
+		.max = FIT_IMG_BASE,
 	},
 	[IMAGE] = {
 		.rr = PCIE_DYN_RR_SRAM_R3,
-		.min = BOOT_FIT_START - 1,
-		.max = BOOT_FIT_END,
+		.min = FIT_IMG_BASE - 1,
+		.max = FIT_IMG_END,
 	},
 	[IMG_END_TO_SRAM_END] = {
 		.rr = PCIE_DYN_RR_SRAM_END,
-		.min = BOOT_FIT_END - 1,
+		.min = FIT_IMG_END - 1,
 		.max = SRAM_END,
 	},
 	[SPSRAM_IMAGE] = {
